apps/condo/domains/ticket/hooks/useTableColumns.tsx

```diff
@@ -242,11 +246,11 @@ export const useTableColumns = (sort: Array<string>, filters: IFilters) => {
                                         </Typography.Text>
                                     )}
                                 />
-                                { ` ${get(property, 'name')}` }
+                                { ` ${formattedFlatNumberMessage} ${unitName}` }
```

Comment:
what about `${text} `? perhaps it should be added

Reply:
text viewed inside Highliter